feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -92,17 +95,30 @@ fun SessionsScreenRoot(
     modifier: Modifier = Modifier,
     viewModel: SessionsViewModel = hiltViewModel(),
     showNavigationIcon: Boolean = true,
-    currentTime: Instant = Clock.System.now(),
     onNavigationIconClick: () -> Unit = {},
     onSearchClicked: () -> Unit = {},
     onTimetableClick: (TimetableItemId) -> Unit = {},
 ) {
     val state: SessionsUiModel by viewModel.uiModel
 
+    val lifecycleObserver = remember(viewModel) {
+        LifecycleEventObserver { _, event ->
+            if (event == Lifecycle.Event.ON_RESUME) {
+                viewModel.onUpdateTimeLine()
```

Comment:
Looks great. But I think we can make the name `onLifecycleResume`. Because it is the event of view.

Reply:
That sounds better!
Fixed to onLifecycleResume.